services_app/src/main/java/org/opendatakit/services/sync/actions/fragments/UpdateServerSettingsFragment.java

```diff
@@ -0,0 +1,152 @@
+package org.opendatakit.services.sync.actions.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.google.android.material.textfield.TextInputLayout;
+
+import org.opendatakit.consts.IntentConsts;
+import org.opendatakit.properties.CommonToolProperties;
+import org.opendatakit.properties.PropertiesSingleton;
+import org.opendatakit.services.R;
+import org.opendatakit.services.sync.actions.activities.VerifyServerSettingsActivity;
+import org.opendatakit.services.sync.actions.viewModels.AbsSyncViewModel;
+import org.opendatakit.services.utilities.UserState;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UpdateServerSettingsFragment extends Fragment {
+
+    private class OnButtonClick implements View.OnClickListener {
+        @Override
+        public void onClick(View v) {
+            if (v.getId() == R.id.btnUpdateServerUrl) {
+                onClickUpdateUrl();
+            } else if (v.getId() == R.id.btnChooseDefaultServer) {
+                inputServerUrl.getEditText().setText(getString(R.string.default_sync_server_url));
+            } else if (v.getId() == R.id.btnVerifyServerUpdateServerDetails) {
+                startVerifyActivity();
+            } else if (v.getId() == R.id.btnScanQrUpdateServerDetails) {
+                // TODO
```

Comment:
Why empty, if it still needs to be implemented please complete or keep proper `Todo` with description.